src/org/tensorflow/yolo/view/ClassifierActivity.java

```diff
@@ -45,11 +45,21 @@
     private BorderedText borderedText;
     private long lastProcessingTimeMs;
 
+// shimatani
+    private String lastRecognizedClass = "";
+    private String nowRecognizedClass = "";
+    private String tts = "";
+    private String msgInf1 = "";
```

Comment:
It is because the descriptive text has become long.
The second line in the image below is `msgInf1` and the third line is `msgInf2`.
https://github.com/keides2/android-yolo-v2/blob/master/sample/%E2%98%85%E7%B2%98%E7%9D%80%E3%83%86%E3%83%BC%E3%83%97%E3%81%A8%E3%83%9C%E3%82%BF%E3%83%B3%E9%9B%BB%E6%B1%A02.png